app/javascript/packages/document-capture/components/selfie-capture.jsx

```diff
@@ -0,0 +1,189 @@
+    'selfie-capture',
+    isCapturing && 'selfie-capture--capturing',
+    value && 'selfie-capture--has-value',
+  ]
+    .filter(Boolean)
+    .join(' ');
+
+  const labelId = `selfie-capture-label-${instanceId}`;
+
+  return (
+    <>
+      <div
+        ref={labelRef}
+        id={labelId}
+        tabIndex={-1}
+        className={['usa-label', isAccessRejected && 'usa-label--error'].filter(Boolean).join(' ')}
+      >
+        {t('doc_auth.headings.document_capture_selfie')}
+      </div>
+      {isAccessRejected && (
+        <span className="usa-error-message" role="alert">
+          {t('doc_auth.instructions.document_capture_selfie_consent_blocked')}
+        </span>
+      )}
+      <div className={classes}>
+        {value ? (
+          <>
+            <div className="selfie-capture__preview-heading usa-file-input__preview-heading">
+              <span />
+              <button
+                type="button"
+                onClick={() => onChange(null)}
+                className="usa-file-input__choose usa-button--unstyled"
+              >
+                {t('doc_auth.buttons.take_picture_retry')}
+              </button>
+            </div>
+            <FileImage file={value} alt="" className="selfie-capture__preview-image" />
+          </>
+        ) : (
+          <>
+            {/* Disable reason: Video is used only for direct capture */}
+            {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+            <video ref={setVideoRef} className="selfie-capture__video" aria-describedby={labelId} />
+            {isCapturing ? (
+              <>
+                <div className="selfie-capture__frame">
+                  <div className="selfie-capture__frame-corner" />
+                  <div className="selfie-capture__frame-corner" />
+                  <div className="selfie-capture__frame-corner" />
+                  <div className="selfie-capture__frame-corner" />
+                </div>
+                <button
+                  type="button"
+                  className="usa-button selfie-capture__capture"
+                  aria-label={t('doc_auth.buttons.take_picture')}
+                  onClick={onCapture}
+                >
+                  <svg
+                    aria-hidden
+                    focusable={false}
+                    role="img"
+                    xmlns="http://www.w3.org/2000/svg"
+                    viewBox="0 0 512 512"
+                    className="selfie-capture__capture-icon"
+                  >
+                    <path
+                      fill="currentColor"
+                      d="M512 144v288c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V144c0-26.5 21.5-48 48-48h88l12.3-32.9c7-18.7 24.9-31.1 44.9-31.1h125.5c20 0 37.9 12.4 44.9 31.1L376 96h88c26.5 0 48 21.5 48 48zM376 288c0-66.2-53.8-120-120-120s-120 53.8-120 120 53.8 120 120 120 120-53.8 120-120zm-32 0c0 48.5-39.5 88-88 88s-88-39.5-88-88 39.5-88 88-88 88 39.5 88 88z"
+                    />
```

Comment:
Can we pull the image into its own component or template? If I had to go update an icon with a newer version, I would not expect to look inside another component like this

Reply:
> Can we pull the image into its own component or template? If I had to go update an icon with a newer version, I would not expect to look inside another component like this

Yeah! That makes total sense. And especially since it'll be quite likely this isn't the only icon we'll ever need access to.

There's a potential can of worms in how best to do that 😅  

- A "Camera" component
   - Or, a parameterized "Icon" component
- With the new package-based architecture, whether it makes sense to l ive in `@18f/identity-document-capture`
   - Or, `@18f/identity-components`
   - Or, a new `@18f/identity-icons`

Parameterized single Icon component could risk becoming unwieldy if left unchecked.

Example: https://github.com/WordPress/dashicons/blob/master/react/index.jsx

I think it might be interesting to at least leave open the possibility of some sort of [code splitting](https://webpack.js.org/guides/code-splitting/) in an implementation, e.g.

```jsx
// components/icon.jsx

// Today:
export default const Icon = {
  Camera: () => <svg { /* ... */ } />,
};

// Possible future:
export default const Icon = {
  Camera: React.lazy(() => import('./icons/camera.jsx')),
};

// In usage: <Icon.Camera />
```

Reply:
That's neat! Yeah this SVG is small, and there aren't any others, so the `// Today` solution seems fine -- or even a solution that eagerly imports from other files so that we have a 1:1 SVG:file ratio

Reply:
Extracted new icon component in 318303f.